suite-native/app/src/StylesProvider.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useColorScheme } from 'react-native';
+
+import { AppColorScheme, selectThemeVariant } from '@suite-native/module-settings';
+import { createRenderer, StylesProvider as TrezorStyleProvider } from '@trezor/styles';
+import { prepareNativeTheme, ThemeColorVariant } from '@trezor/theme';
+import { useDispatch, useSelector } from 'react-redux';
+
+type StylesProviderProps = {
+    children: React.ReactNode;
+};
+
+const renderer = createRenderer();
+
+const DEFAULT_COLOR_VARIANT: ThemeColorVariant = 'chill';
+
+const getColorVariant = (
+    systemColorScheme: ReturnType<typeof useColorScheme>,
+    userColorVariant: AppColorScheme,
+) => {
+    if (userColorVariant !== 'system') {
+        return userColorVariant;
+    }
+
+    switch (systemColorScheme) {
+        case 'dark':
+            return 'dark';
+        case 'light':
+            return DEFAULT_COLOR_VARIANT;
+        default:
+            return DEFAULT_COLOR_VARIANT;
+    }
+};
+
+export const StylesProvider = ({ children }: StylesProviderProps) => {
+    const systemColorScheme = useColorScheme();
+    const userThemeVariant = useSelector(selectThemeVariant);
+
+    const colorVariant = getColorVariant(systemColorScheme, userThemeVariant);
+    const theme = prepareNativeTheme({ colorVariant });
+
+    console.log(useSelector(state => state));
+    console.log(colorVariant, theme);
+    console.log(useDispatch());
```

Comment:
Could you maybe add a comment why it's here? I suppose it's just avoiding eslint to complain but would be nice to write it out so it's screaming into the eyes more and gets deleted soon without forgetting about it.

Reply:
uff this should be removed, some leftover from debugging

Reply:
gone